notebooks/MAST/plotting_quasar_observations/plotting_quasar_observations.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    #Take a peak at the FITS file we downloaded
</pre><p>peak -&gt; peek</p><p> </p><p>The tutorial template prompts to "Explain pertinent details about the file you've just downloaded." so this is a good time to explain the two file extensions and maybe highlight the most relevant/improtant columns in the table. Also that each table entry in the No. 1 HDU is actually an array! I was not expecting that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>